### ISSUE TITLE
confusion_matrix() takes 2 positional arguments but 3 were given

### ISSUE THREAD

**Issue by yanzhao16** [tsunghan-mama/RandLA-Net-pytorch#17]
when i use this code: python3 train_SemanticKITTI.py --checkpoint_path pretrain_model/checkpoint.tar, 
This error will happen:  confusion_matrix() takes 2 positional arguments but 3 were given.
could you tell me how to modify this code?

**Comment by yanzhao16** [comment]
https://github.com/tsunghan-mama/RandLA-Net-pytorch/blob/913837e846176e4247a7e21783bf8f2f38576257/utils/metric.py#L28
